File: frontend/chiro-app/src/app/models/user-short-dto.ts
export interface UserShortDto {
  id: string;
  username: string;
  firstName?: string;
  lastName?: string;
  email?: string;
  phoneNumber?: string;
  dateOfBirth?: string;
  isGroupLeader: boolean;
  profileImageUrl?: string;
  groupId?: string;
}

File: backend/Chiro.Api/Chiro.Application/Dtos/UserShortDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Chiro.Application.Dtos
{
    public class UserShortDto
    {
        public Guid Id { get; set; }
        public string Username { get; set; } = string.Empty;
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string? Email { get; set; }
        public string? PhoneNumber { get; set; }
        public DateTime DateOfBirth { get; set; }
        public bool IsGroupLeader { get; set; }
        public Guid? GroupId { get; set; }
    }
}
